activesupport/lib/active_support/benchmark.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  # = \Benchmark
+  module Benchmark
+    # Benchmark realtime in seconds.
+    #
+    #   ActiveSupport::Benchmark.realtime { User.expensive_query.count }
+    #   # => 5.8273
+    def self.realtime(&block)
```

Comment:
Is there a need for this method? `::Benchmark.realtime` is exactly the same: https://github.com/ruby/benchmark/blob/master/lib/benchmark.rb#L311.

Reply:
The whole point of this PR is that `benchmark` will need to be explicitly be added to the dependencies of rails (because ruby will bundle it). I'm of the oppinion that for what amounts to 3 lines of code is not warranted.

Of course it needs to be added to dependencies right now anyways to suppress the warning, but this PR allows to remove it in future versions.